metrics/web/lib/project_groups/presentation/state/project_groups_notifier.dart

```diff
@@ -47,6 +48,10 @@ class ProjectGroupsNotifier extends ChangeNotifier {
   /// Holds the error message that occurred during updating projects data.
   String _projectsErrorMessage;
 
+  /// Holds the error message that occurred during
+  /// selecting projects on the project group.
+  String _projectSelectionErrorMessage;
```

Comment:
Should we store an instance of the `ProjectGroupProjectIdsValidationErrorMessage` instead of `String` like we are doing it with the `PersistentStoreErrorMessage `?